### PR TITLE
Feature/#567 add possibility to sort courses

### LINKED ISSUE #567: add possibility to sort courses instead of last visited
# User Story

### As a: 
user

### I want:
to have the possibility to sort courses on the dashboard, for example alphabetically.

### so that:
I can find courses more easily.

## Acceptance criteria:
- [ ] courses sortable

## Additional info:
Without sorting the overview of my courses is quite complicated if the number of courses is high enough.
------
_Please tag this issue if you are sure to which tag(s) it belongs._

### PR DESCRIPTION
**Pull request by AlperUygun**
## Description:

Closes #567 
## Improvements
- Courses can now be sorted alphabetically

## Known Issues:
_NONE_

------
_Remember to prefix your PR-Title with `⚠ WIP: ` if you still work on it.
If you have reached a final state, remove the prefix._

### CODE REVIEW

**Comment by PatrickSkowronek** [comment]
The code is good, but the layout is really bad 
screenshot from 2018-04-30 20-13-09

The Sort button should be at the top and you should have the possibility to change it back!

File: app/webFrontend/src/app/start/dashboard/dashboard-student/dashboard-student.component.html [hunk 1]
```diff
@@ -1,2 +1,6 @@
 <app-course-container [title]="'dashboard.myCourses' | translate" [courses]="myCourses" [expand]="true" (onLeave)="leaveCallback()" (onEnroll)="enrollCallback()"></app-course-container>
 <app-course-container [title]="'Available Courses'" [courses]="availableCourses" [expand]="false" (onEnroll)="enrollCallback()"></app-course-container>
+<button mat-raised-button color="primary"
```
[the remark this answers is not in the document]
Comment (AlperUygun) [reply app/webFrontend/src/app/start/dashboard/dashboard-student/dashboard-student.component.html]: Ok

**Comment by AlperUygun** [comment]
Ok, hab jetzt einen mat-select um zwischen den verschiedenen Sortiermöglichkeiten auszuwählen.

File: app/webFrontend/src/app/start/dashboard/dashboard-student/dashboard-student.component.html [hunk 2]
```diff
@@ -1,2 +1,9 @@
+<mat-form-field>
+  <mat-select placeholder="sort options">
+    <mat-option>None</mat-option>
```
Comment (PatrickSkowronek) [inline app/webFrontend/src/app/start/dashboard/dashboard-student/dashboard-student.component.html]: Why is 'none' any option? It should be 'last visited' at standard.

File: app/webFrontend/src/app/start/dashboard/dashboard-student/dashboard-student.component.html [hunk 3]
```diff
@@ -1,2 +1,9 @@
+<mat-form-field>
+  <mat-select placeholder="sort options">
```
Comment (PatrickSkowronek) [inline app/webFrontend/src/app/start/dashboard/dashboard-student/dashboard-student.component.html]: The placeholder should be 'Sorted By'

File: app/webFrontend/src/app/start/dashboard/dashboard-student/dashboard-student.component.html [hunk 4]
```diff
@@ -1,2 +1,9 @@
+<mat-form-field>
```
Comment (PatrickSkowronek) [inline app/webFrontend/src/app/start/dashboard/dashboard-student/dashboard-student.component.html]: The Form should be on the left side!

**Pull request merged**